api/library/python/iterm2/iterm2/filepanel.py

```diff
@@ -89,13 +89,12 @@ def message(self) -> typing.Optional[str]:
     @message.setter
     def message(self, value: str):
         self.__message = value
-        
+
     async def async_run(self, connection) -> 'iterm2.OpenPanel.Result':
```

Comment:
The return type is now `typing.Optional['iterm2.OpenPanel.Result']`

Similar comment for save panel

Reply:
> The return type is now `typing.Optional['iterm2.OpenPanel.Result']`
> 
> Similar comment for save panel

done